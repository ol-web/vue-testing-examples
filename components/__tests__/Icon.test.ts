import { shallowMount } from '@vue/test-utils'

import getMountOptions from '@/tests/helpers/getMountOptions'
import Component from '@/components/Icon.vue'

describe('PrettyTextFromArray', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(Component, getMountOptions({}))

    expect(wrapper.html()).toMatchSnapshot()
  })
})
