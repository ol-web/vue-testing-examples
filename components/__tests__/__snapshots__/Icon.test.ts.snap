// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrettyTextFromArray renders correctly 1`] = `"<svg></svg>"`;
